posthog/tasks/test/test_usage_report.py

```diff
@@ -377,9 +386,9 @@ def _test_usage_report(self) -> List[dict]:
                     "team_count": 2,
                     "teams": {
                         str(self.org_1_team_1.id): {
-                            "event_count_lifetime": 43,
-                            "event_count_in_period": 12,
-                            "event_count_in_month": 32,
+                            "event_count_lifetime": 44,
+                            "event_count_in_period": 13,
+                            "event_count_in_month": 33,
```

Comment:
What do you think about not including these events in the event count? Technically, yes people could use an insight to see how many survey responses they've had, but it doesn't really feel like charging for these events is a value add.

Reply:
Can do, but:

A 1000 survey sent events cost $100 minimum, while the same for product analytics is a rounding error ( ~0.01$?), and at the same time, not adding any overhead to the event count per team query keeps it fast-ish.

No strong feelings here though, happy to defer to what you think is best.

Reply:
although that might change for survey shown / survey dismissed events

Reply:
We can just leave as-is but should do some research into how eg. AWS handles this because it needs to be really clear to customers how dependencies are charged. This is obviously going to keep coming up!